package_name: spirit

camera:
  mock_camera: false  # Default is false.
  show: true  # Default is false.
  streams:
    realtime: true
    slow: true
    past_image: false
  real:
    drone_ip: 192.168.1.1  # Default is 192.168.1.1.
  slowdown: 15  # times. Default is 15.

control:
  use_joystick: true  # Default is true.
  js_number: 1  # Default is 0.

pose:
  mock_pose: false  # Default is false.
  real:
    use_mocap: true  # Default is true.
    use_odometry: false  # Default is false.

past_image:
  eval_method: spirit
  constant_time_delay:
    delay: 2
  constant_distance:
    distance: 1.5
  murata:
    coeff_height: 2
    coeff_direction: 1
    coeff_elevation: 1
    coeff_distance: 4
    coeff_similarity: 0
    z_ref: 0.3
    l_ref: 1.5
  spirit:
    coeff_centred: 2
    coeff_direction: 1
    coeff_distance: 4
    coeff_similarity: 0
    l_ref: 1.5

visualization:
  show: true
  debug: False  # False, online, offline
